analysers/analyser_osmosis_indoor.py

```diff
@@ -40,6 +40,35 @@
     tags != ''::hstore AND
     tags?'indoor' AND
     tags->'indoor' in ('room', 'corridor', 'area', 'level')
+UNION ALL
+SELECT
+    relations.id,
+    outer_ways.linestring AS geom,
```

Comment:
As you use a GROUP BY, you can use here a ST_Collect and a ST_Envelope.

Reply:
:thinking: Is it really useful ? The outer linestring is the exterior ring and `&&` operator does not need it a bbox

Reply:
Not mandatory. The idea is just to keep simplest geom.